packages/cli-exec/src/exec.js

```diff
@@ -58,7 +58,8 @@ export const exec = command('exec', {
     log.warn('Percy is disabled');
   } else {
     try {
-      yield* percy.yield.start();
+      const execType = getExecType(cliCommand);
+      yield* percy.yield.start(execType);
```

Comment:
`yield.start()` is used by multiple commands - upload, snapshot etc 

update that to include `web` as execType

Reply:
Okay if API contract defaults to web then its fine. Please cc someone from API owners to review this part.

Reply:
cc @fotinakis

Reply:
it's our own code we have merged this pr in master https://github.com/percy/percy-api/pull/2958